Lib-DND5e/dnd5e_Property_calculateXPForCR.command

```diff
@@ -0,0 +1,44 @@
+[h: challengeRating = getProperty ("CR")]
+[h, if (challengeRating == ""): challengeRating = 0; ""]
+<!-- I need to test if its outside of bounds, but its dicey -->
+<!-- when the value can be a string or a number -->
+[h, if (isNumber(challengeRating)), code: {
+	[if (challengeRating > 30): challengeRating = 30; ""]
+	[if (challengeRating < 0): challengeRating = 0; ""]
+}]
+[h: crXpMap = json.set ("", 0, 0,
+		"1/8", 25,
```

Comment:
Not the first time I've written a static map in a macro that is a core 5e "table". Look into if it's better to convert to an MT Table and interrogate the table. Allows DMs to later jackify said tables to coerce the campaign to their needs.

Reply:
Tables are too tied to random selection operations. Coercing them into arbitrary key-value lookup tables is a bit too ham-fisty, right now.